### PR TITLE
Add About Us section

### PR DESCRIPTION
**Pull request by Katsuya21**
### Summary

This PR adds the About Us section to the Landing Page.

### Approach

Implemented the component as per the Figma design.

### How to Test the Changes

Checkout the changes at the home page.

### Screenshots or Recordings
image

image

## Checklist

- [ ] I have added/updated tests that cover the changes.
- [ ] I have updated the documentation to reflect the changes.

### CODE REVIEW

**Comment by aps08** [comment]
Will need a live demo

File: packages/ui/src/components/about/About.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        Our product pulls in data from electronic health records (EHRs), practice management systems, and billing software to streamline revenue cycle management.
+      </p>
+      <div className={styles.logoGrid}>
+        {INTEGRATIONS.map((integration) => (
+          <div key={integration.id} className={styles.logoItem}>
+            <img src={integration.src} alt={integration.alt} />
+          </div>
+        ))}
+      </div>
+    </div>
```
Comment (Ayushsanjdev) [inline packages/ui/src/components/about/About.jsx]: I believe all content or text should be in a constant file.

File: packages/ui/src/components/about/About.module.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
+.container {
+  padding: 4rem;
+  max-width: 1200px;
+  margin: 0 0;
+}
+
+.aboutUs {
+  color: #4F46E5;
```
Comment (Ayushsanjdev) [inline packages/ui/src/components/about/About.module.css]: Lets start using reusable color variable names

**Comment by aps08** [comment]
@Katsuya21 conflicts and images not updated.

**Pull request merged**